### PR TITLE
Add redstone mapping names

### PR DESCRIPTION
**Pull request by asiekierka**
Please review!

### CODE REVIEW

File: mappings/none/om.mapping [hunk 1]
```diff
@@ -1,2 +1,2 @@
 CLASS none/om
-	METHOD a getEntityById (I)Ljava/lang/Object;
```
Comment (asiekierka) [inline mappings/none/om.mapping:2]: For the record, this is incorrect, as "om" is a generic.

File: mappings/net/minecraft/block/BlockRedstoneGateBase.mapping [hunk 2]
```diff
@@ -3,8 +3,37 @@ CLASS none/alv net/minecraft/block/BlockRedstoneGateBase
...
 	METHOD <init> (Z)V
 		ARG 0 powered
+	METHOD A isValidInput (Lnone/asj;)Z
+		ARG 0 state
+	METHOD B isGate (Lnone/asj;)Z
+		ARG 0 state
+	METHOD C matchesType (Lnone/asj;)Z
+		ARG 0 other
 	METHOD a getStateForPlacement (Lnone/aiu;Lnone/cn;Lnone/cu;FFFILnone/sn;)Lnone/asj;
+	METHOD a getOutputSignal (Lnone/aiy;Lnone/cn;Lnone/asj;)I
```
Comment (asiekierka) [inline mappings/net/minecraft/block/BlockRedstoneGateBase.mapping]: Should be getOutputLevel... will fix.

**Comment by shadowfacts** [comment]
LGTM

**Comment by asiekierka** [comment]
LGTM, I suppose

**Comment by marcin212** [comment]
LGTM

**Pull request merged**